app/form_objects/support/school_suggestion_form.rb

```diff
@@ -3,19 +3,44 @@ class Support::SchoolSuggestionForm
 
   MAX_NUMBER_OF_SUGGESTED_SCHOOLS = 50
 
-  attr_accessor :name_or_urn, :school_urn
+  attr_accessor :name_or_urn, :school_urn, :except
 
-  validates :name_or_urn, length: { minimum: 3 }
+  validates :name_or_urn, length: { minimum: 3 }, allow_blank: true
```

Comment:
Why do we need the `allow_blank` set to true?

Reply:
It was intended to allow the case where `school_urn` is populated (by the JS-powered auto-complete) and `name_or_urn` is not. I've updated it in dbe5495 to make this clearer.